api/src/opentrons/protocol_runner/protocol_runner.py

```diff
@@ -48,22 +50,50 @@ class ProtocolRunner:
     you will need a new ProtocolRunner to do another run.
     """
 
+    def was_started(self) -> bool:
+        """Whether the runner has been started.
+
+        This value is latched; once it is True, it will never become False.
+        """
+
+    async def load(self, protocol_source: ProtocolSource) -> None:
+        """Load a ProtocolSource into managed ProtocolEngine.
+
+        Calling this method is only necessary if the runner will be used
+        to control the run of a file-based protocol.
+        """
+
+    def play(self) -> None:
+        """Start or resume the run."""
+
+    def pause(self) -> None:
+        """Pause the run."""
+
+    async def stop(self) -> None:
+        """Stop (cancel) the run."""
+
+    async def run(
+        self,
+        protocol_source: Optional[ProtocolSource] = None,
+    ) -> ProtocolRunResult:
+        """Run a given protocol to completion."""
+
+
+class PythonAndLegacyRunner(ProtocolRunner):
+    """Protocol runner implementation for python protocols and legacy protocols."""
```

Comment:
```suggestion
    """Protocol runner implementation for Python protocols, and JSON protocols ≤v5."""
```

Just to be super clear about what we mean by "legacy."